src/sqlancer/postgres/gen/PostgresVacuumGenerator.java

```diff
@@ -24,15 +24,20 @@ public static SQLQueryAdapter create(PostgresGlobalState globalState) {
             sb.append("(");
             for (int i = 0; i < Randomly.smallNumber() + 1; i++) {
                 ArrayList<String> opts = new ArrayList<>(Arrays.asList("FULL", "FREEZE", "ANALYZE", "VERBOSE",
-                        "DISABLE_PAGE_SKIPPING", "SKIP_LOCKED", "INDEX_CLEANUP", "TRUNCATE"));
+                        "DISABLE_PAGE_SKIPPING", "SKIP_LOCKED", "INDEX_CLEANUP", "TRUNCATE", "PARALLEL"));
                 String option = Randomly.fromList(opts);
                 if (i != 0) {
                     sb.append(", ");
                 }
                 sb.append(option);
-                if (Randomly.getBoolean()) {
+                if (option.equals("PARALLEL")) {
```

Comment:
I think it would be better to make it explicit that the "PARALLEL" here and above refer to the same constant. Ideally, I think we would introduce an enum that encodes all the options above.